app/src/main/java/com/jco/taskmaster/MainActivity.java

```diff
@@ -71,7 +71,7 @@ void setupDatabase(){
                 .allowMainThreadQueries()
                 .build();
 
-
+        taskDatabase.taskDatabaseDao().findAll();
```

Comment:
Here, you're finding all the tasks in your database, but you're not assigning them to anything - you'll want to create a list to hold the tasks.